code/modules/mob/living/living.dm

```diff
@@ -288,13 +288,9 @@
 	visible_message("<b>[src]</b> points to [pointed_object]")
 	return TRUE
 
-/mob/living/proc/can_succumb()
-	return health < HEALTH_THRESHOLD_CRIT && stat == UNCONSCIOUS
-
-
 /mob/living/verb/succumb()
 	set hidden = TRUE
-	if(!can_succumb())
+	if(health >= HEALTH_THRESHOLD_CRIT || stat != UNCONSCIOUS)
```

Comment:
you should have a (stat == dead) return before this properly so that the message only shows if you're alive